app/controllers/idv/enter_password_controller.rb

```diff
@@ -1,14 +1,13 @@
 module Idv
   class EnterPasswordController < ApplicationController
-    before_action :personal_key_confirmed
```

Comment:
This... clearly wasn't doing anything, since they shouldn't have confirmed their personal key yet!